tests/Fixture/Tool/ToolWithToolParameterAttribute.php

```diff
@@ -57,7 +57,7 @@ public function __invoke(
             maxProperties: 10,
             dependentRequired: true,
         )]
-        object $shippingAddress,
+        string $shippingAddress,
```

Comment:
why?

Reply:
couldn't find a way to translate that to a meaningful schema.

we can't grab any information about the subitems here - or do you see a way?

Reply:
but it worked before, right?